Source/Urho3D/Network/Connection.cpp

```diff
@@ -73,6 +74,8 @@ Connection::Connection(Context* context, NetworkConnection* connection)
     : AbstractConnection(context)
     , transportConnection_(connection)
 {
+    SetPacketSizeLimit(packedMessageLimit_);
```

Comment:
This is a noop and not necessary. Although `packedMessageLimit_` has a typo and should be `packetMessageLimit_`, maybe this was a source of confusion.

Reply:
> This is a noop and not necessary

How so? This method now also resizes the compression buffers based on the limit

```C++
void Connection::SetPacketSizeLimit(int limit)
{
    packetMessageLimit_ = limit;
    compressedPacketBuffer_.resize(packetMessageLimit_);
    decompressedPacketBuffer_.resize(packetMessageLimit_ * 10);
}
```

> packedMessageLimit_ has a typo and should be packetMessageLimit_

Fixed